src/ocean/meta/traits/Basic.d

```diff
@@ -660,3 +660,38 @@ unittest
 
     static assert(isTypedef!(RealNum) == TypedefKind.Struct);
 }
+
+/******************************************************************************
+
+    Check if the type of a member inside a struct or class is a D manifest
```

Comment:
Why inside a struct or class? Would `bool isManifestConstant ( T )` not be more general?

Reply:
I added another version of this function

Reply:
Is the class or struct member version necessary? Can't you just do `isManifestConstant!(MyStruct.some_field)`?

Reply:
I guess... I removed the prev version and I will update my original code

Reply:
Please ping when this PR is updated.

Reply:
It looks like I need the both functions. The following test fails using `static assert(!isManifestConstant!(Test.init.CustomType));` when `assert(!isManifestConstant!(Test, "CustomType"));` works